docs/howto/error-reporting.txt

```diff
@@ -255,13 +255,6 @@ following methods:
     Returns ``True`` to activate the filtering operated in the other methods.
     By default the filter is active if :setting:`DEBUG` is ``False``.
 
-.. method:: SafeExceptionReporterFilter.get_request_repr(request)
```

Comment:
Deleting this method may be problematic for some use cases. For example, here's a subclass that provides filtering on request.META. As far as I could tell, that code couldn't be adapted to provide filtering after this change. If I'm wrong, the release notes should describe how to update.

https://github.com/blaise-io/storycollective/blob/0378a824984f3753e05235775236d46ee001193e/storycollective/log.py